\documentclass[9pt,twocolumn,twoside]{gsag3jnl}
\usepackage{epstopdf}
\usepackage{hyperref}
\usepackage{setspace}
\usepackage{multirow}
\usepackage{longtable}
\usepackage{caption}
\articletype{mp} % article type
% {inv} Investigations
% {msr} Mutant Screen Reports
% {gs} Genomic Selection
% {goi} Genetics of Immunity
% {gos} Genetics of Sex
% {mp} Multiparental Populations

\captionsetup{justification=raggedright,singlelinecheck=false}
\runningtitle{G3 Journal Template on Overleaf} % For use in the footer

%% For the footnote.
%% Give the last name of the first author if only one author;
% \runningauthor{FirstAuthorLastname}
%% last names of both authors if there are two authors;
% \runningauthor{FirstAuthorLastname and SecondAuthorLastname}
%% last name of the first author followed by et al, if more than two authors.
\runningauthor{Odell \textit{et al.}}

\newcommand{\beginsupplement}{%
        \setcounter{table}{0}
        \renewcommand{\thetable}{S\arabic{table}}%
        \setcounter{figure}{0}
        \renewcommand{\thefigure}{S\arabic{figure}}%
     }

\title{Modeling Allelic Diversity of Multi-parent Mapping Populations Affects Detection of Quantitative Trait Loci}

\author[$\ast$,1,2]{Odell, S. G.}
\author[2,4]{Hudson, A. I.}
\author[3]{Praud, S.}
\author[2,4,5]{Ross-Ibarra, J.}
\author[1]{Runcie, D.}

\affil[1]{Dept. of Plant Sciences, University of California, Davis, CA, USA}
\affil[2]{Dept. of Evolution and Ecology, University of California, Davis, CA, USA}
\affil[3]{Limagrain, Chappes, France}
\affil[4]{Center for Population Biology, University of California, Davis, CA, USA}
\affil[5]{Genome Center, University of California, Davis, CA, USA}

\correspondingauthoraffiliation[$\ast$]{Dept. of Plant Sciences and Dept. of Evolution and Ecology, University of California, Davis, CA, USA
E-mail: sgodell@ucdavis.edu}

\begin{abstract}
The search for quantitative trait loci (QTL) that explain complex traits such as yield and drought tolerance has been ongoing in all crops.
Methods such as bi-parental QTL mapping and genome-wide association studies (GWAS) each have their own advantages and limitations.
Multi-parent advanced generation inter-crossing (MAGIC) populations contain more recombination events and genetic diversity than bi-parental mapping populations and are better able to estimate effect sizes of rare alleles than association mapping populations.
Here we discuss the results of using a MAGIC population of doubled haploid (DH) maize lines created from 16 diverse founders to perform QTL mapping.
We compare three models that assume bi-allelic, founder, and ancestral haplotype allelic states for QTL.
The three methods have differing power to detect QTL for a variety of agronomic traits.
Although the founder approach finds the most QTL, all methods are able to find unique  QTL, suggesting that each model has advantages for traits with different  genetic architectures.
A closer look at a well-characterized flowering time QTL, qDTA8, which contains \emph{vgt1}, highlights the strengths and weaknesses of each method and suggests a potential epistatic interaction.
Overall, our results reinforce the importance of considering different approaches to analyzing genotypic datasets, and shows the limitations of binary SNP data for identifying multi-allelic QTL.
\end{abstract}

\keywords{MAGIC; QTL; linkage mapping; association mapping}

\dates{\rec{July xx, 2021} \acc{xx xx, 2021}}
\DeclareRobustCommand{\rchi}{{\mathpalette\irchi\relax}}
\newcommand{\irchi}[2]{\raisebox{\depth}{$#1\chi$}} % inner command, used by \rchi
\begin{document}

\maketitle
\thispagestyle{firststyle}
%\slugnote
\firstpagefootnote
\vspace{-13pt}% Only used for adjusting extra space in the left column of the first page

\section{Introduction}
\linenumbers
The study of quantitative genetics requires the ability to link differences in phenotype to genotypic variation.
Natural and artificial selection act on phenotypes, but only heritable phenotypic variation will result in changes in population means.
Maize presents an excellent model organism to study quantitative genetics due to the combination of extensive genetic and phenotypic resources and the ability to create mapping populations.
In addition, maize is one of the most widely produced crops in the world and is a major source of calories for millions of people.
Decades of research into maize genetics have resulted in the identification of many quantitative trait loci (QTL) that explain variation in phenotypes such as yield, flowering time, and plant height \citep{Buckler, Wang,Wallace, Beavis, Steinhoff}.
Such traits are extremely agronomically important, but are also crucial in terms of fitness and local adaptation.

Researchers have  discovered large-effect QTL for a number of agronomic traits in maize through the use of different types of mapping populations \citep{Huang2}.
The choice of mapping population comes with associated advantages and limitations.
In particular, different types of populations tend to vary in two main characteristics: (1) their ability to capture genetic diversity and (2) their power to detect QTL of small effect.
Multi-parent Advanced Generation Intercross (MAGIC) populations have been used in breeding to increase the genetic diversity included in a mapping population compared to biparental populations in multiple model organisms. In plants, MAGIC populations have been created for maize \citep{DellAcqua}, wheat \citep{Huang}, tomato \citep{Pascual}, and Arabidopsis \citep{Kover,Huang4}, among others. For animal models, populations such as the Collaborative Cross in mice \citep{Churchill,Aylor} and the \emph{Drosophila} Synthetic Population Resource (DSPR) \citep{King} have likewise been created.
Compared to genome-wide association panels, MAGIC populations have more power to detect low frequency alleles and can better compare allelic effects between founders because the crossing scheme increases the frequency of all parental alleles to be approximately equal.
Simulations of an 8-parent MAGIC population showed that sample sizes of 300 could detect QTL accounting for 12\% of variance with a power of 82\% averaged across minor allele frequencies \citep{DellAcqua}.

In this study, we used a MAGIC population of 344 doubled-haploid lines derived from 16 inbred maize parents developed by the company Biogemma to understand how genetic models can impact the identification of QTL.
Compared to previous populations in maize, this MAGIC population has a greater number of founders, and should have comparable power to larger nested mapping  populations \citep{DellAcqua,Yu2}. In addition, using doubled haploids (DHs) instead of recombinant inbred lines (RILs) removes any residual heterozygosity, ensuring that replicates are genetically identical.
For these reasons, the Biogemma MAGIC population that we present here has great potential to reveal new insights into the genetic control of quantitative traits in maize.

\begin{figure*}[ht!]
\centering
\includegraphics[width=\textwidth,height=10cm]{figures/Fig1A_v2.png}
\caption{\textbf{The crossing scheme of the MAGIC population.}}
\label{fig:figure1a}
\end{figure*}

In addition to the choice of mapping population, the choice of how to model allelic variation can impact the power of a study to detect and analyze QTL.
A bi-allelic model for QTL, often used in GWAS, assumes that the underlying causal variants for QTL are explained by two alleles, usually SNPs, that are segregating in the population.
%Most commonly, genetic variation is represented as bi-allelic SNPs which segregate in the population, with each individual possessing either a reference or alternate allele.
With this bi-allelic model, hereafter referred to as $GWAS_{SNP}$, markers represent a small region of the chromosome that is in tight linkage disequilibrium with the genotyped SNP.

An alternative model for the allelic state of QTL can be used in multi-parent populations, where we assume that each founder contributes its own allele.
In this model, comparable to interval mapping, rather than looking at individual SNPs, an allele becomes the founder identity of that region, or which parent of the population that segment of chromosome was derived from.
As a result, QTL are multi-allelic, with the number of alleles equal to how many founders were used in the making of the population.
We will refer to this founder model hereafter as $QTL_F$.

The two allelic models described above make the assumption that for each genotyped SNP, there are either two distinct alleles in the population ($GWAS_{SNP}$) or as many alleles as there are founders ($QTL_F$).
The latter assumption, although very possible for biparental mapping populations, becomes increasingly unlikely as the number of founders increases.
This is because the founders used in the making of a population are related to one another with varying degrees of distance, and therefore, most likely share ancestral haplotypes through identity-by-descent (IBD).
A third allelic model takes into account shared ancestral haplotypes between founders.
This model, hereafter referred to as $QTL_H$, allows the number of alleles at each site to vary anywhere from two to the total number of founders (here 16), based off of the number of ancestral haplotypes at that site.
This has the potential to increase statistical power compared to the $QTL_F$ model by reducing the number of parameters that must be estimated by the model.

Here we present a maize MAGIC population derived from 16 parents and discuss the performance of three different models for representing allelic states: bi-allelic, founder, and ancestral haplotype allelic models for detecting QTL.
Using \emph{vgt1}, a well-characterized flowering time QTL with a strong candidate causal variant that is variable in the population, we demonstrate differences between the three methods and explore potential interactions between \emph{vgt1} and other genetic variation in the population.

\section{Materials and Methods}
\label{sec:materials:methods}
\subsection{Mapping Population}
The MAGIC population was derived from 16 inbred maize parents representing the diversity of temperate maize (\href{run:./figures/Methods_Supplementa1.docx}{Supplemental File 1}).
The parents were chosen to be a mix of members of Flint and Dent heterotic groups, which are historically and genetically diverged groups that, when crossed, produce F1 plants that display hybrid vigor.
In addition, the founders were chosen to represent a phenotypically diverse set of flowering times for temperate maize.
The 16 founder lines were crossed in a funnel crossing scheme, and then the resulting synthetic population was intercrossed for 3 generations with around 1600 individuals per cycle (Figure \ref{fig:figure1a}) (\href{run:./figures/Methods_Supplementa1.docx}{Supplemental File 1}).
Finally, 800 lines were selected from the synthetic population to create doubled haploids (DH), resulting in 550 MAGIC DH lines at the end of the process.
The MAGIC DH lines were crossed to a tester MBS847 to produce 344 hybrids (Figure \ref{fig:figure1a}).

\subsection{Genotype Data}
The 16 founder lines and the MAGIC DH lines were all genotyped with the Affymetrix 600K Axiom SNP array \citep{Unterseer}, resulting in genotype data for 517,769 SNPs.
A  total of 503,902 SNPs from were used from the  600K after filtering out invariant sites and sites that were not located on autosomal chromosomes according to the B73 AGPv4 reference genome \citep{Jiao}.
Linkage disequilibrium was calculated as $R^2$ of pairwise SNPs using the software PLINK and the 600K SNP data (Figure \ref{fig:supfigure10}) \citep{Purcell}.
%To estimate the rate of LD decay, for the purpose of computational ease, a random subset of 6,000 SNPs were selected, and the pairwise $R^2$ value of this subset was calculated against all SNPs \.

\subsection{Phenotype Data}
The MAGIC F1 plants were phenotyped in five different field locations in four different years, resulting in six distinct environment-years (Hudson et al. in prep).
The environments represent a range of latitudes and water stress, from vegetative and flowering water deficit (Nerac, 2016) to optimum well-watered conditions (Graneros, 2015).
In each environment we grew a minimum of 292 and a maximum of 309 of the DH lines. Each genotype was grown with two replicates in each environment.
In all environments, six traits were measured: grain yield (GY), plant height (PH), female flowering date (DTS), male flowering date (DTA), thousand kernel weight (TKW), and harvest grain moisture (HGM). By subtracting male flowering date from female flowering date we also obtained anthesis-silking interval (ASI).
For each of the lines we calculated best linear unbiased predictor (BLUP) scores for all seven phenotypes, combining measurements from all environments to get estimates of the genetic contribution to the phenotype for each MAGIC line \citep{Aulchenko}.

\begin{figure*}[h!]
\centering
\includegraphics[width=\textwidth,height=11cm]{figures/Methods_Fig1.png}
\caption{\textbf{Founder representation of the MAGIC population}(\textbf{A}) Coverage of each founder across the population on chromosome 10 as a percentage. Black line represents expected number of lines per founder with equal distribution (6.25\%). Each of the 16 founders are indicated by a color. (\textbf{B}) Total coverage of each founder across the population as a percentage. Black line shows expectation of equal distribution (6.25\%)  (\textbf{C}) Founder probabilities for 10 MAGIC DH lines on chromosome 10 in genetic distance.}
\label{fig:figure1}
\end{figure*}

\subsection{Calculation and Validation of Founder Probabilities}
We used the package R/qtl2 \citep{Broman} to determine founder probabilities of the MAGIC DH lines using the 600K genotype data and the cross type ``riself16''.
We then filtered down the founder probabilities from all 503,902 sites to represent founder recombination blocks for interval mapping.
Markers for the $QTL_F$ approach were filtered based on linkage disequilibrium using an iterative approach where a marker was dropped if the $R^2$ value of the founder probabilities between it and the previous marker was greater than 0.95.
After filtering, a total of 4,578 sites were kept to represent founder recombination blocks in the MAGIC DH lines.

Due to the fact that the actual crossing scheme and the cross type input into R/qtl2 differed (DH lines rather than RILs), we wanted to assess the accuracy of the founder probabilities.
This was done by simulating lines using the actual crossing scheme and assessing the performance of the calc\_genoprobs function of R/qtl2 in correctly identifying the founder genotype (Figure \ref{fig:figure1}C).
We developed an R package \citep{R}, \emph{magicsim} (\url{https://github.com/sarahodell/magicsim}) to simulate the lines using the maize consensus genetic map from \citep{Ogut} to generate approximate recombination rates across the chromosome.
We simulated 100 MAGIC populations constituting 325 lines and assessed founder assignment accuracy as the average percentage of SNPs where the predicted founder was the same as the actual founder.

\subsection{Test for Equal Representation of Alleles}
The number of lines that had a probability of a particular founder greater than 0.8 were used as  an approximation of the number of lines that had that founder at a site.
This observed count was compared to a null expectation of $1/16$ for equal distribution across lines (approximately 21 lines per founder)(Figure \ref{fig:figure1}B \& C).
We performed a $\rchi^2$ test for each site to determine if founder counts significantly deviated from null expectation.
We obtained a 5\% significance threshold using the $\rchi^2$ obtained from founder counts in 100 simulated populations.
The $\rchi^2$ tests from simulated lines were done using reconstructed genotype data pulled from the 600K genotype data of the 16 founders.
We then used the same methods of calculating founder probabilities with R/qtl2 used with the actual population.
Due to the fact that we used the inferred founder identities of the simulated lines, rather than the known founder identities, the null distribution of p-values generated from $\rchi^2$ tests of the simulated populations incorporated uncertainty of founder assignment.

\begin{figure*}[ht]
\includegraphics[width=\textwidth,height=10cm]{figures/figure2.png}
\caption{\textbf{Diversity and Size of Haplotype Blocks} The number of unique haplotypes among the 16 MAGIC founders across the 10 chromosomes of maize (\textbf{A}) The number of unique haplotypes per haplotype block and size of haplotype blocks along chromosome 4 in physical distance. (\textbf{B}) Distribution of unique haplotypes per haplotype block across the genome. (\textbf{C}) Distribution of haplotype block size in physical distance, represented as log10(bp). The black bar represents the average size of 303.7 kb. (\textbf{D}) Distribution of haplotype block size in genetic distance, represented as cM. The black bar represents the average size.}
\label{fig:figure2}
\end{figure*}

\subsection{Calculation of Identity-by-Descent and Haplotype Probabilities}
The identification of regions of shared genetic sequence between founder pairs allows collapsing of founders into haplotypes.
IBD was measured from the 600K SNP data of the founders using the software RefinedIBD with a sliding window of 10 cM and a minimum IBD segment length of 0.2 cM \citep{Browning}.
The resulting segments of pairwise IBD between each of the 16 founders were used to identify distinct haplotype blocks.
We did this by moving along the chromosome, starting a new haplotype block when a segment of pairwise IBD between founders started or ended (Figure \ref{fig:supfigure6}).
Then, within blocks, we grouped all founders that were in IBD with one another into a haplotype.
Within blocks, the founder probabilities for founders that shared a haplotype were summed to obtain haplotype probabilities.

In certain instances, the pairs of founders that were in IBD with one another in a particular haplotype block formed an incomplete graph, where  not all founders were in IBD with all other founders (Figure \ref{fig:supfigure6}).
For example, from the results of RefinedIBD, founder A was in pairwise IBD with both founder B and founder C, but founder B and C were not in pairwise IBD.

For the sake of simplicity, we continued with the assumption that all founders in a haplotype were in IBD with one another (we called B and C as in IBD).
However, it is important to note that cutoffs for IBD are inherently arbitrary, and haplotypes called here do still possess genetic differences between founders, with some founders more different than others.

Markers for the $QTL_H$ mapping approach were filtered for LD using an iterative approach similar to $QTL_F$: for all haplotype blocks with the same number of distinct haplotypes, a marker was dropped if the correlation of haplotype probabilities between it and the previous marker was greater than 0.95.
After filtering, a total of 11,105 sites were kept to represent haplotype blocks in the MAGIC DH lines.

\subsection{Association and QTL Mapping}
The R package GridLMM \citep{Runcie} was used to run association mapping using the three different models of representing the QTL allelic state.
The function GridLMM\_ML was used with the "ML" option.
The following three models were approximated by fitting each locus independently.
The three methods differed in the $X$ matrix used in the mixed linear model.
The bi-allelic model ($GWAS_{SNP}$) was

\begin{equation}
\label{eqn:gridlmm1}
 y = x_{Si}{\beta_{Si}} + Zu + \epsilon
\end{equation}

where $y$ is the response variable, $x_{Si}$ is an $n$ x 1 genotype vector for SNP $i$ with reference and alternate alleles represented as 0 and 1, respectively, $\beta_{Si}$ is the effect size of the alternate allele, $Z$ is the design matrix, $u \sim \mbox{N}(0,\sigma^2_u \bf{K})$ is the random effects of markers across the rest of the genome using the genomic relationship matrix, $K$, and $\epsilon$ is the error.
The genomic relationship matrix, $K$ was generated from the 600K SNP data using a leave-one-chromosome-out (LOCO) method.
The same $K$ matrix was used for all three models.
GridLMM fit an $n$ x 1 matrix for each site $p$.
The founder model ($QTL_F$):
\begin{equation}
\label{eqn:gridlmm2}
 y = X_{Fi}{\beta_{Fi}} + Zu + \epsilon
\end{equation}
where $X_{Fi}$ is a $n$ x $f-1$ matrix for marker $i$ and $x_{fni}$ is the probability that at site $i$, individual $n$ was derived from founder $f$, and $\beta_Fi$, is the effect size of each founder allele. GridLMM fit an $n$ x $f$ matrix for each site $p$.
The ancestral haplotype model ($QTL_H$):
\begin{equation}
\label{eqn:gridlmm3}
 y = X_{Hi}{\beta_{Hi}} + Zu + \epsilon
\end{equation}
where $X_{Hi}$ is an $n$ x $h-1$ matrix for marker $i$ and $x_{hni}$ is the probability that at site $i$, individual $n$ has haplotype $h$, and $\beta_{Hi}$, is the effect size of each haplotype allele. GridLMM fit an $n$ x $h$ matrix for each site $p$.
Significance cutoffs for p-values were obtained using permutation testing, taking the 5\% cutoff from 1000 permutations where genotypes were randomized relative to phenotypes for each method.
For the two largest QTL peaks, \emph{qDTA3-2} and \emph{qDTA8}, we tested if the peaks contained more than one QTL by including the most significant SNP of the QTL as a covariate and testing to see if any SNPs within the support interval remained significant.

 \subsection{Model Comparison}
The results of the three models were compared using two main criteria: (i) presence or absence of identified QTL peaks and (ii) the size of QTL support intervals.
QTL support intervals were determined by identifying the most significant SNP for a QTL peak and demarcating the left and right bounds of the QTL as the left-most and right-most SNPs within a 100Mb window centered on the highest SNP that have a -log10(p-value) that is 2 log10(p-values) below that of the highest SNP.
The detection of QTL was compared across the three methods for each phenotype.
A QTL was said to be identified across models if the QTL support interval for that QTL overlapped.
The effect of the model used on the size of QTL support intervals was investigated using the QTL which were identified by all three methods (n=26).
The support interval size response variable was represented both in terms of physical distance (Mb) and genetic distance (cM).

\subsection{Estimation of Effect Sizes}
We used the R package \emph{lme4qtl} to calculate standard errors of effect sizes relative to the population mean \citep{Ziyatdinov}.
For the $QTL_F$ model, effect sizes were dropped for individual founders at some sites if there where fewer than 5 MAGIC lines that had a probability greater than 0.8 for one of the founders.
This same filtering was done with $QTL_H$ effect sizes for sites with low representation of particular haplotypes.
This was to ensure that effect sizes for individual founders and haplotypes could be effectively estimated.
We confirmed that effect sizes calculated by GridLMM and lme4qtl matched one another, with the correlations in effect sizes between the two methods greater than 0.99.

\subsection{Tests for Epistasis}
We ran a genome scan for epistatic interactions with \emph{vgt1}.
The probability of the MAGIC lines having the MITE insertion at \emph{vgt1} was calculated by summing the founder probabilities for all founders that have the $MITE^+$ allele at the site closest to the location of the MITE underlying \emph{vgt1} in the B73 APGv4 assembly found on MaizeGDB \citep{Portwood}.
Lines that had uncertain allelic states at the MITE (0.05 > Pr($MITE^+$) < 0.95) were dropped for the test.
Applying a Bonferroni significance threshold adjusted for the number of tests, we tested for epistasis using the 600K genotype data.

We also performed QTL mapping with $QTL_F$ using only the $MITE^+$ MAGIC lines.
This was to see if there were any other loci whose effect was only observed in the presence of the MITE.
A normal epistatic model could not be fit with founder alleles because there were not enough degrees of freedom to compare each founder.
We used the model from Equation \ref{eqn:gridlmm2} using DTA BLUP scores and the 5\% significance threshold for DTA.

\subsection{Flowering Time Enrichment Tests}
We used a list of flowering time (FT) genes assembled by \citep{Li5} to test for enrichment of FT genes in (i) regions that had significantly uneven representation of the founders and (ii) regions with high inter-chromosomal LD ($R^2$ > 0.9).
Of the 907 genes, we used 887 which were aligned to chromosomes 1 through 10 in the B73 AGPv4 assembly \citep{Jiao}.
To determine a null distribution, we randomly sampled 887 non-FT genes and counted the number that overlapped with these two sets of regions for 1,000 permutations.
We compared this distribution to the actual number of FT genes that overlapped with the selected regions.

\subsection{Data availability}
Genotypic data will be made available through FigShare.
Phenotypic and environmental data will be made available at FigShare associated with our companion paper, \citep{Hudson_in_prep}(tracking number G3-2021-402688) (https://figshare.com/s/5ee8337defdef63b04ce). Supplemental files are available at FigShare. File S1 contains a detailed description of the crossing scheme used to develop the MAGIC population. File S2 shows Manhattan plots similar to Figure \ref{fig:figure3} for individual environments. File S3 shows founder effect size plots for \emph{vgt1} similar to Figure \ref{fig:foundervgt1figure} for individual environments. File S4 is contains tables of all QTL identified in the study, their locations, and their effect sizes estimates from the three models. Code used to run analyses and to generate simulated data can be found at \url{https://github.com/sarahodell}.

\section{Results}

\subsection{MAGIC Population}
We developed a 16-parent MAGIC population using temperate inbred maize lines representative of the diversity of the Flint and Dent heterotic groups of North America and Europe (Figure \ref{fig:figure1a}).

We genotyped 344 MAGIC DH lines from the population with a 600K SNP genotyping array and measured 7 phenotypes across 6 environments from hybrids resulting from crossing the DH lines to an inbred tester, MBS847.
Using the phenotype data from the six environments, we calculated BLUP scores for each of the MAGIC lines.
PCA Analysis of the MAGIC lines and the 16 founders and tester suggested that the MAGIC lines maintained much of the genetic variation possessed by the founders, without overwhelming bias towards any particular founders (Figure \ref{fig:supfigure7}).
In addition, the minor allele frequencies of SNPs in the 16 founder compared to in the MAGIC lines suggested that lower frequency SNPs in the founders were either maintained or brought up in frequency in the MAGIC lines, which aids in the estimation of SNP effect sizes (Figure \ref{fig:supfigure8}).

\subsection{Simulation and Validation of Founder Probabilities}

We partitioned the genomes of individual MAGIC lines into segments of ancestry from the 16 founders.
This allowed us to determine the predicted contribution of each founder to the population (Figure \ref{fig:figure1}C).
The founder probabilities determined using R/qtl2 were able to assign founders to the actual MAGIC DH lines with high confidence (>0.80) for  96.7 \% of the 10 chromosomes of maize.
The median size of recombination blocks was 4.32 Mb and the mean size was 15.76 Mb with a standard deviation of 29.52 Mb.
The average number of crossover events per line was 123.7 with a standard deviation of 20.73.
Our simulations suggest a very high ($\mu = 99.8\%, \sigma =0.011$) assignment accuracy (see Methods).
This reinforced our confidence in the founder probabilities obtained from the actual data.

\subsection{Identity-By-Descent and MAGIC Haplotypes}
In some cases, the model which inferred founder identity in the MAGIC lines had high uncertainty, with probabilities split approximately equally between two founders.
We hypothesized that this uncertainty was due to the two founders having very similar genetic sequence at those regions, such that the model struggled to differentiate the two.
To assess the genetic similarity of the founders, we calculated pairwise Identity-By-Descent (IBD) between all founders using the software RefinedIBD \citep{Browning}.
Areas of uncertainty in founder probabilities of the DH lines were associated with regions of IBD between two or more founder lines in that region of the chromosome (Figure \ref{fig:supfigure5}).

The results showed that a total of 1.81 Gb (86\%) and 1367.5 cM (92.7\%) of the genome were in IBD between at least two different founders.
The average size of an IBD segment between two founders was 140kb (0.51 cM) with a median of 122kb (0.46 cM). Pairwise IBD segments sizes ranged from 8 kb (0.3 cM) to  673 kb (1.61 cM).
For founder pairs that were found to be in IBD with one another, the total percentage of IBD between founders ranged from 0.0018\% (F492 and VA85) to 4.39\% (B73 and A632), with an average of 0.061\%.
There were no IBD segments found for 18 of 120 possible pairwise founder combinations.
The amount of IBD segments between the 16 founders and the tester, MBS847, was mostly low (ranging from 0.14\% of the genome for F2 to 3.7\% for B73), with the notable exception of DK63, which was in IBD with MBS847 for 36.5\% of the genome.
The Neighbor-Joining Tree of the relatedness of the 16 founders and MBS847 recapitulated the IBD results (Figure \ref{fig:supfigure9}).
For a particular founder pair, B73 and A632, there were large segments where the lines shared haplotypes, and the tree placed them very close together.
This is consistent with pedigree of the lines, where A632 was derived from B14, a line from the same heterotic group as B73 (\href{run:./figures/Methods_Supplementa1.docx}{Supplemental File 1}) \citep{Lorenz}.

Due to the widespread Pairwise IBD between the founders, it appeared that many founders shared ancestral haplotypes.
Within individual blocks of ancestry, we collapsed founder alleles that were identical by descent into a single haplotype (see Methods).
The genome was broken up into a total of 6,929 haplotype blocks.
Of those blocks, approximately 16\% of them (1,152) contained at least one haplotype whose pairwise IBD between parents was incomplete, meaning that there was some genetic variation between founders within those haplotypes that was not captured by the haplotype designation (see Methods)(Figure \ref{fig:supfigure6}).
The number of unique haplotypes within haplotype blocks varied across chromosomes, ranging from 6 at the lowest to 16 at the highest (Figure  \ref{fig:figure2}A \& B).
The average number of unique haplotypes per haplotype block was 13 ($\mu = 12.85, \sigma =1.71$) (Figure \ref{fig:figure2}B).
There was a wide range of haplotype block sizes, with the average physical size of haplotype blocks being 303.7 kb ($\sigma =1.71$Mb) (Figure \ref{fig:figure2}C)).
The largest haplotype block was 39.3 Mb long on chromosome 7, which had 16 unique haplotypes.
In genetic distance, haplotype block sizes range from 0 to 3.4 cM, with an average of 0.20 cM and a median of 0.11 cM (Figure \ref{fig:figure2}D).

\begin{figure}
\includegraphics[width=\columnwidth,height=18cm]{figures/ft_interchrom_ld_figure.png}
\captionsetup{width=.9\columnwidth}
\caption{\textbf{Inter-chromosomal Linkage Disequilibrium and Flowering Time Genes in the MAGIC Population} \textbf{(A)} Ribbons represent regions of $R^2$ > 0.9 between consecutive SNPs on different chromosomes. Dark, solid, red bands outlined in black are regions larger than 10Mb on at least on of the chromosomes. Lighter, translucent red bands are regions greater than 5Mb and less than 10Mb. Red lines are regions larger than 100kb and smaller than 5Mb. Grey lines are regions larger than 1kb. The blue regions represent the support intervals for the flowering time QTL, \emph{qDTA3-2} and \emph{qDTA8}. \textbf{(B)} The null distribution of the number of genes overlapping with regions of high interchromosomal-LD ($R^2$ > 0.9) from 800 randomly selected genes from 1000 permutations. Observed number of flowering time genes overlapping with regions of high interchromosomal-LD are shown in red (n=593).}
\label{fig:circosfigure}
\end{figure}
\subsection{Founder Representation and Linkage Disequilibrium}

Analysis of the MAGIC population showed that the overall representation of the 16 founders in the MAGIC DH lines was relatively even, with the highest percentage founder, A654, representing 6.7\% and the lowest percentage founder, EP1, representing 5.2\%, compared to the expectation of 6.25\% for each founder (Figure \ref{fig:figure1}B).
However, multiple chromosome regions deviated significantly from the expected equal distribution (Figure \ref{fig:figure1}A).
Across individual regions of each chromosome, 20.4\% of the genome significantly deviated from null expectations compared to 100 simulated populations (5\% significance threshold $\rchi^2$ test p-value < 1.5e-09) (Figure \ref{fig:supfigure11}).
The fact that the $\rchi^2$ test performed on 100 simulated MAGIC populations of 344 individuals resulted in far fewer significant $\rchi^2$ peaks shows that the over- and under-representation of certain founder alleles was greater than would be expected by chance.
It also shows that over- and under-representation of founder alleles in the population was not due to potential inaccuracy of R/qtl2 in assigning founders.
These results suggests that a large amount of the over- and under-representation of founder alleles in the MAGIC population is biological, rather than a result of model error, and perhaps evidence of selection for or against particular founder alleles.

We estimated recombination rates and linkage disequilibrium across the MAGIC lines.
The intra-chromosomal LD structure shows fast LD decay consistent with many recombination events (Figure \ref{fig:supfigure10}).
Unexpectedly, there was a large amount of high inter-chromosomal LD (Figure \ref{fig:circosfigure}A).
Of a total of 9,796,630 SNP pairs with an $R^2$ greater than or equal to 0.9, 426,178 (4.3\%) of those pairs came from different chromosomes.
The number of inter-chromosomal high LD regions was more than would be expected by chance:
in 100 simulated populations, there were no SNP pairs with $R^2$ greater than 0.9 detected between chromosomes.
We detected large segments of inter-chromosomal LD between chromosome 3 and chromosome 8, but these regions did not overlap with the support intervals for \emph{qDTA8} and \emph{qDTA3-2}, corresponding to \emph{vgt1} and \emph{vgt3}, respectively (Figure \ref{fig:circosfigure}A).

Because individual DH lines were required to overlap in flowering time with the tester, MBS847, in order to successfully make F1s for the MAGIC population (Figure \ref{fig:figure1a}) , we hypothesized that the patterns observed in founder representation and linkage disequilibrium might be due to selection on flowering time.
In particular, the tester, MBS847, is a generally later flowering line and is $MITE^-$ at \emph{vgt1} \citep{Chardon}, providing the opportunity for selection against early flowering alleles.
As one test of this hypothesis, we asked whether genes involved in flowering were enriched in regions with high inter-chromosomal LD, and found strong enrichment of flowering time genes  in those regions compared to 1,000 randomized permutations (p-value<0.001)(\ref{fig:circosfigure}B).
We found no significant enrichment of flowering time genes within regions with significantly uneven founder representation (Figure \ref{fig:supfigure4}).
The finding that regions with high LD between chromosomes contain more flowering time genes than would be expected by chance suggests that selection for the combination of particular alleles affecting flowering may have occurred.

\begin{figure*}[h!p]
\centering
\includegraphics[width=\textwidth,height=22cm]{figures/Methods_Fig3_v2.png}
\caption{\textbf{Results of three methods of QTL identification} (\textbf{A-C}) Manhattan plots for Days to anthesis (DTA) BLUPs. Colored points represent significant QTL above the 5\% significance threshold from 1000 random permutations, located between log10(p-value)=5 and log10(p-value)=6 (dashed line). \textbf{(A)} GWAS results using the $GWAS_{SNP}$ method. \textbf{(B)} Results from QTL mapping the $QTL_F$ method \textbf{(C)} Results from QTL mapping using the $QTL_H$ method. (\textbf{D-F}) Identification of individual QTL with varying permutation significance thresholds of 1\%, 5\%, and 10\% across the three methods. Different colors indicate the phenotype of the QTL and darker colors indicate that a QTL was significant at that threshold for a particular method for $GWAS_{SNP}$ (\textbf{D}), $QTL_F$, (\textbf{E}), and $QTL_H$ (\textbf{F}).}
\label{fig:figure3}
\end{figure*}

\subsection{QTL Mapping and Association Mapping}
We performed association mapping using three models of the allelic state of QTL.
The first method, $GWAS_{SNP}$, used SNP genotypes obtained from the 600K array, assuming that QTL are bi-allelic.
The second method, $QTL_F$, used probabilities of founder identity in chromosome intervals, assuming that a QTL had as many alleles as founders.
The third method, $QTL_H$, used probabilities of haplotype identity in chromosome intervals, assuming that a QTL had as many alleles as ancestral haplotypes.
We performed QTL mapping separately in each of the 42 environment:phenotype combinations, plus the across-environment averages of the 7 traits, for a total of 49 separate analyses.
The three methods varied in their ability to identify QTL.
The majority of QTL were identified by all three models using a 5\% significance threshold (26 or 57\%), with 6 QTL found in both $QTL_F$ and $QTL_H$ and 1 QTL found in both $GWAS_{SNP}$ and $QTL_F$ (Table \ref{tab:suptable2}).
In addition, each model found unique QTL, with 7, 3, and 3 QTL found in only $GWAS_{SNP}$, $QTL_F$, and $QTL_H$, respectively (Figure \ref{fig:supfigure2}).

Next we merged QTL of the same phenotype from different environments based on overlapping support intervals.
There were 20 unique across-environment QTL identified from all methods at a 5\% significance threshold, and 10 of these  were found in more than one environment.
We found 12 across-environment QTL  using BLUPs and 2 of these were not identified in any individual environment.
Of these across-environment QTL, 6 (0 unique) QTL were found in Blois, 2014, 7 (1 unique) in Blois, 2017, 7 (3 unique) in Graneros, 2015, 5 (1 unique) in Nerac, 2016, 6 (3 unique) in St.Paul, 2017, and 3 (0 unique) in Szeged, 2017.

Figure \ref{fig:figure3}A-C shows the Manhattan plots from the three methods for BLUP DTA.
Analysis with individual environments identified fewer QTL, but displayed similar patterns (\href{run:./figures/Methods_Supplemental2.pdf}{Supplemental File 2}).
For DTA, all three methods easily identify two large QTLs, \emph{qDTA3-2} on chromosome 3 and \emph{qDTA8} on chromosome 8.
These QTL correspond to three previously identified QTL, \emph{vgt3} for \emph{qDTA3-2} and \emph{vgt1} and \emph{vgt2} for \emph{qDTA8}.
In addition, there are multiple QTL that are found by only a subset of the models.
Using BLUPs, the $GWAS_{SNP}$ method was able to identify one QTL on chromosome 5 for thousand-kernel weight, \emph{qTKW-5} that was not found in either $QTL_F$ or $QTL_H$.
A second QTL on chromosome 3 for DTA, \emph{qDTA3-1}, was only found with the $QTL_H$ method.
A DTA QTL on chromosome 9, \emph{qDTA9} and a harvest grain moisture QTL on chromosome 3, \emph{qHGM3-1} were only found using $QTL_F$.
However, a QTL for DTS with overlapping support intervals to \emph{qDTA9} was found in both $QTL_F$ and $QTL_H$.

Despite differences in the models, the power to identify and refine the location of QTL was similar across the three methods.
$QTL_F$ was able to identify the most QTL, regardless of changes in the significance threshold (Figure \ref{fig:supfigure3}).
Individual QTL that were found in one method at the 5\% significance threshold usually became significant in other methods when the 10\% threshold was used, indicating that the differences in the ability to detect these QTL between methods is mostly due to differences in power (Figure \ref{fig:figure3}D-F).
Nonetheless, there were multiple QTL that were identified in only one method. There were 10 QTL that appeared in only one method, mostly related to grain yield, plant height, and thousand kernel weight traits.

Support intervals for QTL were determined using a twofold change in the log$_{10}$ p-value of the most significant SNP (see Methods).
The size of $GWAS_{SNP}$ support intervals were significantly larger than $QTL_H$ support intervals in genetic distance (1.55 cM, SE = 0.56, t-ratio = 2.77, p-value = 0.021), but the difference in physical distance was not significant (t-ratio = 1.97, p-value = 0.13)(Table \ref{tab:suptable1}).
There was no significant difference in the size of QTL support intervals between $QTL_F$ and $QTL_H$ in either physical and genetic distance.
Although on average, the physical and genetic size of $GWAS_{SNP}$ support intervals were larger than those of $QTL_F$ support intervals, the difference was not significant, perhaps because of a single outlier QTL in $QTL_F$ with a very large support interval (Figures \ref{fig:supfigure15} \& \ref{fig:supfigure16}).
When the outlier was dropped from the model, the difference between $GWAS_{SNP}$ and both $QTL_F$ and $QTL_H$ support intervals were significant in both genetic (t-ratio = 2.87, p-value = 0.017; t-ratio = 3.17, p-value = 7.4e-3) and physical distance (t-ratio = 2.91, p-value = 0.015; t-ratio = 2.68, p-value = 0.027).

\begin{figure*}[ht!]
\centering
\includegraphics[width=17cm,height=9cm]{figures/vgt1_Figure4.png}
\caption{\textbf{Estimated founder effect sizes for \emph{vgt1}.} Estimates of founder effect sizes relative to the population mean for Days to Anthesis (DTA) BLUP phenotypes using $QTL_F$. Letters A-N and groupings of shaded blocks along the x-axis indicate founders that have shared haplotypes at the most significant $QTL_F$ SNP. Color indicates reference (0) and alternate (1) alleles at the most significant $QTL_F$ SNP. Positive effect sizes indicate later flowering and negative effect sizes indicate earlier flowering. Founders on the right (D-N) within the bracket are lines that possess the MITE insertion ($MITE^+$), while founders on the left (A-C) lack the MITE insertion ($MITE^-$).}
\label{fig:foundervgt1figure}
\end{figure*}

\subsection{Variation around \emph{vgt1}}
One notable QTL that was identified by all three models using BLUPs (Figure \ref{fig:figure3}) and nearly all individual environments was qDTA8, a large QTL on chromosome 8 that was strongly correlated with variation in days to anthesis as well as days to silking.
The support interval for this QTL overlapped with two previously characterized flowering time QTL, \emph{vgt1} and \emph{vgt2}.
As a well-studied, large effect QTL, \emph{vgt1} provides a useful benchmark for comparison of the three allelic models.

At \emph{vgt1}, the 16 founders are segregating ($MITE^+$/$MITE^-$) for the putative causal variant, a MITE insertion in a conserved non-coding sequence upstream of \emph{ZmRap2.7} \citep{Salvi,Castelletti}.
Four founders, B73, OH43, VA85, and B96, lack the MITE insertion, while the other 12 are $MITE^+$.
Looking at the most significant SNP for \emph{qDTA8} from $GWAS_{SNP}$, the alternate allele correlated imperfectly with the presence of the MITE in the founders (r=0.65).
We expected to see $QTL_F$ effect sizes at this locus that match the allelic state of the founders, with $MITE^+$ founders having earlier effect sizes and $MITE^-$ founders having later effect sizes.
However, for some founders the $QTL_F$ effect sizes at \emph{vgt1} deviated from those expectations (Figure \ref{fig:foundervgt1figure}).
Four $MITE^+$ founders, A632, F252, C103, and F492, had DTA BLUP effect size estimates later than the population average.
While only F252 had a 95\% confidence interval not overlapping zero, all  had effect sizes significantly later than the other $MITE^+$ founders (t-ratio = 7.67, p-value <1e-4).
This pattern was also seen in the effect sizes estimated in individual environments (\href{run:figures/Methods_Supplemental3.pdf}{Supplemental File 3}).
Lastly, at the most significant hit from $QTL_H$, founders are grouped into haplotypes consistent with their allele at the MITE, but there are still far more than 2 distinct haplotypes (14).
Analysis of the haplotype structure in the region around \emph{vgt1} in the 16 founders showed clear differences between those that do and do not have the MITE insertion, but did not differentiate $MITE^+$ late founders from $MITE^+$ early founders (Figure \ref{fig:supfigure14}).

One possible explanation for this observation is an epistatic interaction between \emph{vgt1} and other loci in the genome.
However, a genome scan for epistasis between \emph{vgt1} and other loci did not yield any significant interactions (Figure \ref{fig:supfigure12}).
$QTL_F$ using only MAGIC lines predicted to have the MITE had two significant DTA BLUP QTL in the region of \emph{vgt1} (Figure \ref{fig:supfigure13}).
One of these significant sites is located in close proximity to the causal gene for \emph{vgt2}, \emph{ZCN8}, and may be explained by this linked QTL \citep{Guo}.
The second significant site is located ~15Mb downstream of \emph{vgt1}, suggesting that some local variation around the region of \emph{vgt1} impacts the effect of the QTL on flowering time.
This site may have an epistatic interaction with \emph{vgt1} that did not pass the stringent genome-wide significance threshold.
Alternatively, the relationship between the loci could be entirely additive, but the causal allele may only occur on the $MITE^+$ background.

\section{Discussion}

We used three models of QTL allelic states to identify QTL in the MAGIC population, a bi-allelic model ($GWAS_{SNP}$), a founder multi-allelic model ($QTL_F$), and an ancestral haplotype mult-allelic model ($QTL_H$).
The $GWAS_{SNP}$ method should be most powerful at identifying QTL for which the causal variant is bi-allelic and the tagged SNP is in tight LD with the causal variant.
However, for multi-allelic QTL or QTL for which LD is low between tagged SNPs, this method should have lower power.
$QTL_F$, which assumes that all founders possess distinct alleles, increases the odds of detecting both QTL that are multi-allelic and QTL whose causal variant is not in tight LD with any one tagged SNP.
And while the higher number of parameters that must be fit by this model may also reduce power because the regions tested are much larger than one SNP, it also reduces the multiple testing burden.
Lastly, $QTL_H$ potentially improves on the power of $QTL_F$ to detect QTL that meet the above criteria by reducing the number of parameters that must be estimated.
There is, however, the potential for $QTL_H$ to obscure the signal of some QTL if founders are called as having the same ancestral haplotype when they actually differ for a causal variant.
Due to the fact that $QTL_H$ and $QTL_F$ only take into account recent recombination events, whereas $GWAS_{SNP}$ uses historical recombination, we predicted that $GWAS_{SNP}$ would result in higher resolution of QTL support intervals.
Higher resolution QTL are ideal in that they make it easier to narrow down candidate genes and potential causal variants.

The results of using these three models of genetic architecture to identify QTL suggest that each has its own advantages and disadvantages in terms of how many and which QTL they can identify.
Overall, the $QTL_F$ model performed the best in terms of the number of QTL identified, although there were multiple QTL identified uniquely in all models.
The larger size of $GWAS_{SNP}$ support intervals was unexpected, as this method is often used to fine map QTL regions identified by linkage mapping.
We suspect that this finding is either the result of the somewhat naive method used to determine QTL support intervals, or residual long-distance LD caused by the funnel crossing scheme.
A two-fold drop in log$_{10}$ p-values heavily penalizes QTL that just pass significance.
In addition, the 100Mb window used is blind to QTL peaks that contain multiple QTL, although. the two largest QTL peaks identified, \emph{qDTA3-2} and \emph{qDTA8} each appeared to only contain one QTL.
As a result, it is difficult to determine a reason for the difference in support interval size between the models.

Previous studies have used variations of these methods to identify QTL, and some have directly compared them.
The use of combined linkage and association analyses, sometimes referred to as linkage disequilibrium - linkage association (LDLA) was first proposed by \cite{Meuwissen}, who used predicted IBD probabilities between parents using an evolutionary model and applied them to linkage mapping.
LDLA has been used in multiple studies to enhance QTL detection in multi-parent populations in maize \citep{Giraud2,Yu2,McMullen} and other organisms such as pigs \citep{Herault}, rice \citep{Zaw,Stadlmeier}, barley \citep{Sannemann}, and winter wheat \citep{Stadlmeier}.
\cite{Jansen} used a haplotype-based method for QTL mapping and showed through simulation that this strategy could reduce the number of estimated parameters and, therefore, increase power.
\cite{Yalcin} compared a bi-allelic model to an ancestral haplotype model in a multi-parent mouse population, as well as a "merge analysis" to attempt to identify candidate causal variants underlying QTL.
A similar analysis in the outbred NIH-HS rat population used merge analysis and found that many QTL could not be explained by a single causal variant, suggesting a multi-allelic basis for these QTL \citep{Baud}.

%Simulations of power in the mice Collaborative Cross \citep{Valdar}
Different means of determining ancestral haplotype blocks from parental sequences have been used, with clusthaplo \citep{Leroux}, an extension of the software MCQTL \citep{Jourjon}, being a commonly used algorithm in recent studies.
Bayesian frameworks have also been implemented in real \citep{PerezEnciso} and simulated \citep{Bink} multi-parent populations.
\cite{Giraud} used both a haplotype- and founder- based approach in two nested association mapping populations of Northern European flint and dent maize lines created by \cite{Bauer} and genotyped with a 50K SNP array \citep{Ganal}.
\cite{Giraud} used clusthaplo to determine haplotype blocks based on IBD between parents and used discrete founder and haplotype values in their models.

Previous studies have compared similar models differing in a few notable ways.
The type of multi-parent population used in studies has varied across studies.
To our knowledge, comparison of the performance of these models in a MAGIC population has not been done.
In addition, our method of generating haplotype blocks in $QTL_H$ is distinct from previous work.

Interestingly, the performance of bi-allelic, founder, and ancestral haplotype models differs across studies.
\cite{Giraud} found that their haplotype model outperformed the founder and SNP models in terms of the number of QTL identified using EU-NAM Flint and Dent maize populations.
In contrast, \cite{Garin2} found that in the EU-NAM Flint population, the bi-allelic model detected a larger number of unique QTL, compared to parental or ancestral haplotype models.
\cite{Bardol} found that in two multi-parent dent populations, their bi-allelic model and ancestral haplotype model generally outperformed the parental linkage model, although benefits of these models varied by dataset.
The performance of the three models seems to depend heavily on the diversity of the parents used to generate the population.
For populations with more diverse founders, it would be expected that there would be fewer shared haplotypes between founders, reducing the efficacy of a haplotype model \citep{Giraud}.
The fact that the $QTL_F$ model outperformed the $QTL_H$ and $GWAS_{SNP}$ in our population suggests that the MAGIC population contains a relatively more diverse representation of temperate maize than populations used in previous studies.
It is also possible that the structure of multi-parent populations has an effect on the performance of the three models, compared to previous studies which used nested association mapping \citep{Giraud,Garin2} and factorial populations \cite{Bardol}.

Differences in the estimated effect sizes across models offer suggestions as to the reason for their differences in QTL detection.
QTL that were only found in the $GWAS_{SNP}$ method most likely have a bi-allelic causal variant.
It is likely that the increased number of parameters in the $QTL_F$ and $QTL_H$ models reduce statistical power when the true number of functional alleles is low.
Similarly, we predict multi-allelic QTL were more likely to be identified by the $QTL_F$ or $QTL_H$ models and not the $GWAS_{SNP}$ method unless the effect size was large.
For QTL that were identified in the $QTL_H$ method and not the $QTL_F$ method, there tended to be a lower number of unique ancestral haplotypes, suggesting that $QTL_H$ was more successful in finding these QTL due to improved power when there were fewer functional alleles than founders.
For QTL that were not identified by  the $QTL_H$ method, particularly for QTL that were successfully identified by $QTL_F$, there are two possible explanations.
Because there were more regions tested for $QTL_H$ than for $QTL_F$, if the true number of ancestral haplotypes at the QTL is large, the $QTL_H$ method may actually have lower power than $QTL_F$ because the number of tests is higher.
Alternatively, it may be the result of a failure of $QTL_H$ to accurately represent the true haplotype structure of the QTL region.

On the whole, most QTL were found by all three methods, so there was limited ability to draw reliable conclusions about underlying mechanisms that caused the methods to perform better or worse.
Generally, the comparison of QTL detection and effect size estimates suggested that the methods failed and succeeded on a QTL-by-QTL basis.
This is to be expected, as each QTL is the result of a distinct set of one or more causal alleles with a unique evolutionary history and pattern of linkage disequilibrium within the population.

Whether QTL that appeared in only one method are due to false positives or true differences in the methods' abilities to identify QTL with different genetic architectures cannot be determined, but many of the QTL identified in the MAGIC have underlying candidate genes or have been found in previous studies, providing support to their biological reality.
Multiple flowering time QTL support intervals overlap or are close to previously identified flowering time genes and QTL.
\emph{qDTA9} is nearby the previously identified maize flowering time gene \emph{ZmCCT9} \citep{Huang3}.
\emph{qDTA3-2} overlaps with \emph{vgt3}, whose underlying gene was identified as \emph{ZmMADS69} \citep{Liang}.
\emph{qDTA3-1} is nearby a recently identified flowering time QTL also associated with phasphatidylcholine levels \citep{RodriguezZapata}.
The support interval for \emph{qDTA8} overlaps with two flowering time QTL, \emph{vgt1}, which we discuss in length, and another, \emph{vgt2}.
The causal gene for \emph{vgt2} is \emph{ZCN8}, which is the maize ortholog of FT in \emph{Arabidopsis} \citep{Lazakis}.
Variation in the promoter region of \emph{ZCN8} between temperate maize and teosinte suggests that earlier flowering alleles were under selection during the process of maize domestication \citep{Guo,Bouchet}.
It is interesting to note that there is strong overlap in the support intervals of QTL found on chromosome 3 between flowering time and harvest grain moisture (Table \ref{tab:suptable2}), perhaps due to developmental pleiotropy linking flowering time and the moisture of kernels at harvest.
Three GxE QTL were detected in this population \citep{Hudson_in_prep} using the $QTL_F$ model, but none overlapped with the main effect QTL we detected in this study.
One of these QTL appeared to be a false positive resulting from low representation of one of the founders in the region, indicating the potential for low founder sample size to confound QTL results.

Due to the fact that the MAGIC population is segregating for \emph{vgt1}, it provides an opportunity to further study the mechanism behind the QTL's affect on flowering time.
One benefit of using founder and haplotype approaches lies in the potential to dissect the effects of individual founders and haplotypes within QTL.
This allowed us to look more closely at \emph{vgt1} and observe an interesting pattern of effect sizes that deviated from our expectations based on previous research.
Previous research  has shown that variation in flowering time at this site is strongly correlated with a MITE insertion about 70kb upstream of the flowering time regulator, \emph{ZmRAP2.7}, an \emph{APETALA}-like transcription factor, with the presence of the MITE associated with an earlier flowering time \citep{Castelletti}.
Within maize heterotic groups, Flint maize lines tend to possess the early-flowering allele of \emph{vgt1} (\emph{MITE+}), while dents (such as B73) tend to carry the late-flowering allele (\emph{MITE-}) \citep{Salvi}.
In addition to being a crucial agronomic trait, flowering time contributes to local adaptation for annual plants such as maize, ensuring that individuals can reproduce within the growing season of their environments.
The frequency of the MITE in maize populations follows a latitudinal gradient, suggesting that the early $MITE^+$ allele was selected for during the process of maize adaptation to temperate climates \citep{Navarro}.
It has also been shown that there are differentially-methylated regions around \emph{vgt1} between B73, landrace maize, and its wild relative, teosinte \citep{Xu}.
The hypothesized mechanism of action is that the MITE represses expression of the negative flowering time regulator,\emph{ZmRAP2.7}, possibly due to changes in methylation around the insertion, resulting in earlier induction of flowering \citep{Castelletti}.
However, the MITE has not yet been experimentally shown to result in earlier flowering.
A recent study using multiple multi-parent populations suggested that variation in the effect of \emph{vgt1} in different genetic backgrounds was due to local genetic variation surrounding \emph{vgt1}, rather than epistasis with distant loci \citep{Rio}.
The observed lack of significant epistasis with \emph{vgt1} in this study (Figure \ref{fig:supfigure11}), combined with our results of $MITE^+$ $QTL_F$ (Figure \ref{fig:supfigure12}), appear consistent with this idea.
This finding suggests two possibilities: either (1) that the causal variant underlying \emph{vgt1} is some as-yet unidentified variant that is in tight, but imperfect linkage disequilibrium with the MITE insertion, or (2) that the MITE insertion directly impacts flowering time, and that another variant nearby has a modifying effect on the MITE.
This opens up new areas of inquiry for future studies. \citep{Mathew} used the barley population of \cite{Sannemann} to detect epistatic interactions in flowering time using a Bayesian framework, and a similar method could perhaps be applied here.

The population displayed unexpected patterns of uneven founder representation along the genome (Figure \ref{fig:figure1}C).
The apparent high over- or under-representation of some founder alleles may be due to these founders being in close IBD with another founder, resulting in uncertainty in the founder probabilities.
Nonetheless, the high accuracy of founder assignment and low number of sites that significantly deviate from equal founder distribution in our simulations make this explanation seem unlikely.
If we assume that the simulated populations are accurate representations of the construction of the MAGIC population without selection, this result would suggest that the differences in founder representation observed in the actual population may be due to selection for or against certain founder alleles.

Another interesting observation of the population was the relatively high levels of inter-chromosomal LD (Figure \ref{fig:circosfigure}), which deviated significantly from that obtained from simulations.
A potential consequence of inter-chromosomal LD is the chance for confounding of association analyses, namely resulting in the detection of "ghost" QTL.
Although for the discussed QTL detected that were in high inter-chromosomal LD with one another, \emph{qDTA8} and \emph{qDTA3-2}, their effects on flowering time were independent, the chance for false positives and inaccurate support intervals due to LD structure is still worth noting.
Inter-chromosomal LD has been detected in multiple populations of domesticated organisms, where breeding has resulted in the preservation of certain combinations of favorable alleles between chromosomes \citep{Robbins,MalyshevaOtto}.
Strong selection and positive or negative epistasis in natural populations have also been shown to create a pattern of inter-chromosomal LD \citep{Kulminski,Gupta,Hench,Petkov}.
Both of these observations suggest that forces other than those of random segregation have operated on the MAGIC population, further supported by the fact that genes affecting flowering time were strongly enriched in regions of high inter-chromosomal LD (Figure \ref{fig:circosfigure}).
These observations raise the possibility that the complex crossing schemes of MAGIC populations have the potential to introduce the influence of selection, which may restrict the very genetic variation that we are attempting to study.

\section{Conclusion}
The MAGIC population presented here provides a useful resource for investigating quantitative trait variation in temperate maize.
As a multi-parent population, it has the advantages of increased genetic diversity and higher power to detect QTL with lower allele frequencies.
Simulations of the MAGIC population provide an opportunity to validate assignment of founder identities, as well as generate null expectations for various aspects of the population.
Overall, we find that a founder multi-allelic model identifies the most QTL, although all three models of allelic state are effective at identifying QTL.
The benefits of increasing statistical power by reducing model parameters in bi-allelic and ancestral haplotype models seem to be tempered by the true allelic complexity of the multi-parent population being studied.
We conclude that if the goal of a study is to find as many QTL as possible, then it would be most useful to consider the genetic diversity contained within the population when choosing a model in order to maximize QTL identification.


\section{Acknowledgments}
We would like to acknowledge the Limagrain breeding support and trialing team for producing the doubled haploids and running the field trials used in this study.

\section{Funding}
S.G.O. was supported by the UC Davis Dept. of Plant Sciences and NSF grant 1754098. A.I.H. was supported by National Science Foundation Graduate Research Fellowship 1650042. J.R.I. was supported by USDA Hatch project CA-D-PLS-2066-H 548. D.E.R. was supported by USDA Hatch project 1010469 and USDA NIFA grant no. 2020-67013-30904.

\section{Conflicts of interest}
We state that we have no conflict of interest.

\bibliography{magic_tex}
\beginsupplement
\onecolumn
\section*{Supplement}

\begingroup[ht]
  %\footnotesize
  %\setlength\LTleft{-20pt}%
  %\setlength\LTright{-30pt}%
  \centering
\begin{longtable}{c c c c c c c c}
\caption{\textbf{QTL Identified by Method and Environment} \label{tab:suptable2}} \\
%\renewcommand{\familydefault}{\sfdefault}\normalfont
%\begin{tabularx}{\textwidth}{xxxxxXXx}
\hline \hline
Chr. & ID & Phenotype & Method & Environment & Support Interval (bp) & Threshold\\
\hline
%chr 1
1  & qTPH1 & PH & $QTL_F$ & BLUP & 13,053,862:15,379,189 & 10\% \\
&  qTKW1 & TKW1 & $QTL_F$ & Szeged (2017) & 11,671,250:26,014,128 & 10\% \\
&  qGYD1-1 & GY & $GWAS_{SNP}$ & St. Paul (2017) & 63,241,728:71,780,754 & 5\% \\
&  qGYD1-2 & GY & $GWAS_{SNP}$ & St. Paul (2017) & 173,166,714:173,401,292 & 5\% \\
\hline
%chr2
2  & qTKW2 & TKW & $QTL_F$ & Blois (2017) & 17,689,185:20,386,177 & 10\% \\
&  qTKW2 & TKW & $QTL_H$ & Blois (2017) & 19,213,370:28,310,359 & 10\% \\
&  qHGM2 & HGM & $QTL_F$ & Graneros (2015) & 207,356,837:210,899,385 & 5\% \\
\hline
%chr 3
3 &  qDTA3-1 & DTA & $GWAS_{SNP}$ & BLUP & 9,576,867:20,032,599 & 10\% \\
&  & & & Blois (2017) & 9,977,725:54,966,584 & 10\% \\
&  & & $QTL_F$ & BLUP & 9,468,968:11,692,672 & 10\% \\
&  & & & Blois (2017) & 9,468,968:11,692,672 & 5\% \\
&  & & $QTL_H$ & BLUP & 10,774,922:11,444,207 & 5\% \\
&  & & & Blois (2017) & 9,812,175:11,444,207 & 5\% \\
&  & & & Nerac (2016) & 10,774,922:11,444,207 & 10\% \\
&  qDTS3-1 & DTS & $QTL_H$ & BLUP & 10,774,922:11,444,207 & 10\% \\
&  & & $QTL_H$ & Nerac (2016) & 10,774,922:11,444,207 & 5\% \\
&  qTKW3 & TKW & $QTL_F$ & Blois (2017) & 12,160,355:12,556,111 & 10\% \\
&  qHGM3-1 & HGM & $GWAS_{SNP}$ & Blois (2104) & 16,215,776:34,201,739 & 5\% \\
&  & & $QTL_F$ & BLUP & 12,160,355:31,807,621 & 5\% \\
&  & & $QTL_H$ & BLUP & 10,774,951:54,007,246 & 10\% \\

&  qDTA3-2 & DTA & \multirow{1}{4em}{$GWAS_{SNP}$} & BLUP & 156,128,886:161,296,968 & 1\% \\
&  & & & Blois (2017) & 129,315,698:179,001,343 & 10\% \\
&  & & & Graneros (2015) & 156,128,886:161,412,773 & 1\% \\
&  & & & Nerac (2016) & 153,057,659:163,216,372 & 1\% \\
&  & & & St. Paul (2017) & 123,607,128:161,296,968 & 5\% \\
&  & & \multirow{1}{4em}{$QTL_F$} & BLUP & 155,999,962:161,420,983 & 1\% \\
&  & & & Blois (2014) & 155,999,962:161,420,983 & 1\% \\
&  & & & Blois (2017) & 155,999,962:163,307,287 & 1\% \\
&  & & & Graneros (2015) & 155,999,962:163,307,287 & 1\% \\
&  & & & Nerac (2016) & 158,563,749:161,475,295 & 1\% \\
&  & & & St. Paul (2017) & 155,999,962:161,475,295 & 1\% \\
&  & & & Szeged (2017) & 141,032,560:161,238,626 & 10\% \\
&  & & \multirow{1}{4em}{$QTL_H$} & BLUP & 155,852,367:155,852,367 & 1\% \\
&  & & & Blois (2014) & 157,175,061:161,412,773 & 1\% \\
&  & & & Blois (2017) & 157,175,061:161,412,773 & 1\% \\
& & & $QTL_H$ & Graneros (2015) & 159,141,941:161,486,117 & 1\% \\
&  & & & Nerac (2016) & 160,207,590:161,435,844 & 1\% \\
\hline \hline
Chr. & ID & Phenotype & Method & Environment & Support Interval (bp) & Threshold\\
\hline
3 & qDTA3-2 & DTA & $QTL_H$ & St. Paul (2017) & 155,852,367:161,481,215 & 1\% \\
&  qDTS3-2 & DTS & \multirow{1}{4em}{$GWAS_{SNP}$} & BLUP & 141,622,242:161,296,968 & 1\% \\
&  & & & Blois (2014) & 140,531,962:163,216,283 & 1\% \\
&  & & & Graneros (2015) & 156,128,886:161,296,968 & 1\% \\
&  & & & Nerac (2016) & 160,214,408:168,342,144 & 1\% \\
&  & & & Szeged (2017) & 141,622,242:163,294,109 & 1\% \\
&  & & \multirow{1}{4em}{$QTL_F$} & BLUP & 160,273,248:161,420,983 & 1\% \\
&  & & & Blois (2014) & 140,531,962:163,216,283 & 1\% \\
&  & & & Blois (2017) & 141,032,560:163,307,287 & 1\% \\
&  & & & Graneros (2015) & 153,465,886:163,307,287 & 1\% \\
&  & & & Nerac (2016) & 160,273,248:161,420,983 & 1\% \\
&  & & & Szeged (2017) & 141,032,560:161,420,983 & 1\% \\
&  & & $QTL_H$ & BLUP & 155,852,367:163,307,287 & 1\% \\
&  & & & Blois (2014) & 159,604,235:161,412,773 & 1\% \\
&  & & & Blois (2017) & 141,153,604:163,307,287 & 1\% \\
&  & & & Graneros (2015) & 152,190,251:161,435,844 & 1\% \\
&  & & & Nerac (2016) & 160,392,864:161,237,016 & 1\% \\
&  & & & Szeged (2017) & 137,673,677:161,412,773 & 1\% \\
&  qHGM3-2 & HGM & $GWAS_{SNP}$ & BLUP & 133,870,185:157,564,855 & 1\% \\
&  & & $QTL_F$ & BLUP & 102,059,129:163,726,450 & 5\% \\
&  & & $QTL_H$ & BLUP & 135,632,376:157,563,421 & 5\% \\
&  qTPH3 & PH & $GWAS_{SNP}$ & BLUP & 139,282,905:183,874,792 & 10\% \\
&  & & & Nerac (2016) & 167,424,633:174,758,506 & 1\% \\
&  & & $QTL_F$ & Nerac (2016) & 168,869,189:171,846,046 & 10\% \\
&  & & $QTL_H$ & BLUP & 149,051,759:183,807,330 & 10\% \\
\hline
% chr4
4  & qGYD4 & GY & $GWAS_{SNP}$ & Nerac (2016) & 174,745,869:174,758,681 & 10\% \\
\hline
%chr 5
5  & qTKW5 & TKW & $GWAS_{SNP}$ & BLUP & 1,721,566:1,723,167 & 5\% \\
& GYD5 & GYD & $GWAS_{SNP}$ & St. Paul (2017) & 18,196,101:19,207,809 & 5\% \\
\hline
%chr 6
6  & qGYD6 & GY & $GWAS_{SNP}$ & BLUP & 30,749,664:61,643,265 & 10\% \\
&  & & $QTL_F$ & BLUP & 26,946,128:50,840,937 & 10\% \\
&  qTPH6 & PH & $QTL_F$ & Blois (2014) & 105,707,110:106,378,416 & 10\% \\
\hline
%chr 7
7  & qDTA7 & DTA & $GWAS_{SNP}$ & Graneros (2015) & 15,490,475:20,506,774 & 5\% \\
&  & & $QTL_F$ & Graneros (2015) & 9,361,661:17,862,397 & 5\% \\
&  & & $QTL_H$ & Graneros (2015) & 9,361,661:17,937,993 & 10\% \\
& qTKW7-1 & TKW & $GWAS_{SNP}$ & Blois (2017) & 10,938,777:27,939,165 & 5\% \\
&  & & $QTL_F$ & Blois (2017) &  10,973,468:22,700,741 & 1\% \\
&  & & $QTL_H$ & Blois (2017) &  10,933,860:82,800,018 & 5\% \\
&  qTPH7 & PH & $GWAS_{SNP}$ & BLUP & 9,548,103:10,085,223 & 1\% \\
&  & & & Blois (2017) & 9,378,259:10,085,223 & 5\% \\
&  & & $QTL_F$ & BLUP & 8,991,474:10,085,337 & 1\% \\
\hline \hline
Chr. & ID & Phenotype & Method & Environment & Support Interval (bp) & Threshold\\
\hline
7 & qTPH7 & TPH & $QTL_H$ & BLUP & 8,979,716:10,085,223 & 5\% \\
&  qTKW7-2 & TKW & $GWAS_{SNP}$ & Graneros (2015) & 124,347,596:124,756,367 & 1\% \\
&  & & $QTL_F$ & Graneros (2015) & 95,797,065:131,824,096 & 5\% \\
&  & & $QTL_H$ & Graneros (2015)  & 107,149,522:131,212,353 & 10\% \\
\hline
%chr 8
8  & qDTA8 & DTA & $GWAS_{SNP}$ & BLUP & 126,570,432:151,224,199 & 1\% \\
&  & & & Blois (2014) & 126,885,610:137,874,699 & 1\% \\
&  & & & Blois (2017) & 126,570,432:151,224,199 & 1\% \\
&  & & & Graneros (2015) & 120,341,014:146,112,050 & 1\% \\
&  & & & Nerac (2016) & 120,341,014:151,891,941 & 5\% \\
&  & & & St. Paul (2017) & 126,413,801:151,224,199 & 1\% \\
&  & & & Szeged (2017) & 126,885,610:151,891,941 & 1\% \\
&  & & $QTL_F$ & BLUP & 126,077,002:150,520,256 & 1\% \\
&  & & & Blois (2014) & 126,910,275:137,947,309 & 1\% \\
&  & & & Blois (2017) & 127,378,541:150,866,243 & 1\% \\
&  & & & Graneros (2015) & 120,340,882:150,520,256 & 1\% \\
&  & & & Nerac (2016) & 119,704,031:150,866,243 & 1\% \\

&  & & & St. Paul (2017) & 126,077,002:150,866,243 & 1\% \\
&  & & & Szeged (2017) & 142,750,019:152,574,587 & 1\% \\
&  & & $QTL_H$ & BLUP & 125,337,514:149,986,813 & 1\% \\
&  & & & Blois (2014) & 128,420,026:137,415,945 & 1\% \\
&  & & & Blois (2017) & 126,910,275:150,924,566 & 1\% \\
&  & & & Graneros (2015) & 119,775,957:149,437,688 & 1\% \\
&  & & & Nerac (2016) & 119,100,585:151,722,943 & 5\% \\
&  & & & St. Paul (2017) & 126,910,275:149,986,813 & 1\% \\
&  & & & Szeged (2017) & 128,742,959:151,006,728 & 1\% \\

&  qDTS8 & DTS & $GWAS_{SNP}$ & BLUP & 120,341,014:151,224,199 & 1\% \\
&  & & & Blois (2014) & 126,885,610:135,966,917 & 1\% \\
&  & & & Blois (2017) & 145,540,028:151,224,199 & 1\% \\
&  & & & Graneros (2015) & 119,781,443:137,874,699 & 1\% \\
&  & & & Szeged (2017) & 126,885,610:151,875,955 & 1\% \\
& & & $QTL_F$ & BLUP & 126,077,002:150,520,256 & 1\% \\
&  & & & Blois (2014) & 126,077,002:136,360,604 & 1\% \\
&  & & & Blois (2017) & 146,136,754:150,346,508 & 1\% \\
&  & & & Graneros (2015) & 119,704,031:136,360,604 & 1\% \\
&  & & & Szeged (2017) & 127,992,851:150,866,243 & 1\% \\

&  & & $QTL_H$ & BLUP & 127,378,541:149,437,688 & 1\% \\
&  & & & Blois (2014) & 128,420,026:136,008,951 & 1\% \\
&  & & & Blois (2017) & 145,263,274:149,986,813 & 1\% \\
&  & & & Graneros (2015) & 119,336,430:136,835,221 & 1\% \\
&  & & & St. Paul (2017) & 125,337,514:151,006,728 & 10\% \\
\hline \hline
Chr. & ID & Phenotype & Method & Environment & Support Interval (bp) & Threshold\\
\hline
8 & qDTS8 & DTS & $QTL_H$ & Szeged (2017) & 127,378,541:151,006,728 & 1\% \\
&  qTPH8 & PH & $QTL_F$ & Graneros (2015) & 135,165,296:168,996,496 & 10\% \\
\hline
%chr 9
9  & qTKW9 & TKW & $GWAS_{SNP}$ & BLUP & 15,122,815:18,089,022 & 10\% \\
&  & & $QTL_H$ & BLUP & 15,656,271:18,533,708 & 10\% \\
9  & qDTA9 & DTA & $QTL_F$ & BLUP & 111,962,632:121,725,096 & 5\% \\
&  & & & Blois (2014) & 111,015,021:120,670,613 & 10\% \\
&  & & & Graneros (2015) & 114,343,703:118,040,833 & 10\% \\
&  & & & St. Paul (2017) & 111,962,632:124,483,594 & 5\% \\
&  & & $QTL_H$ & BLUP & 111,015,458:150,056,511 & 10\% \\
&  & & & St. Paul (2017) & 111,962,632:124,460,017 & 5\% \\
&  qDTS9 & DTS & $QTL_F$ & BLUP & 118,060,423:121,725,096 & 1\% \\
&  & & & Blois (2014) & 116,613,531:124,089,472 & 1\% \\
&  & & $QTL_H$ & BLUP & 114,817,909:124,763,331 & 5\% \\
&  & & & Blois (2014) & 114,347,350:124,860,954 & 5\% \\
\hline
10 & qASI10 & ASI & $GWAS_{SNP}$ & BLUP & 116,610,226:123,356,331 & 10\% \\
\hline
%\end{tabularx}
\end{longtable}
\endgroup


\begin{table*}[htbp]
\renewcommand{\familydefault}{\sfdefault}\normalfont
\centering
\caption{\bf Size of QTL Support Intervals By Model}
\begin{tabularx}{\textwidth}{XXXXXXX}
\hline
\header Model &  Avg. Size (Mb) & Median Size (Mb) & Std. Dev (Mb) & Avg. Size (cM) & Median Size (cM) & Std. Dev (cM)\\
\hline
$GWAS_{SNP}$ & 17.53 & 19.67 & 10.19 & 7.31 & 8.21 & 3.50 \\
$QTL_F$ & 14.99 & 10.28 & 13.64 & 6.22 & 5.88 & 3.78 \\
$QTL_H$ & 13.72 & 9.24 & 10.27 & 5.75 & 5.45 & 3.63 \\
\hline
\end{tabularx}
  \label{tab:suptable1}
\end{table*}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/all_chroms_ld_decay.png}
\caption{\textbf{Linkage Disequilibrium of the MAGIC population by Chromosome} Average $R^2$ of pairwise SNPs broken into bins of 100kb by distance.}
\label{fig:supfigure10}
\end{figure}

\begin{figure}[ht]
\centering
    \includegraphics[width=16cm,height=10cm]{figures/IBD_graph_example.png}
\caption{\textbf{Complete and Incomplete IBD Graphs}. \textbf{(left)} An example of the creation of haplotype blocks. The presence of pairwise IBD between two founders along the chromosome is shown as blue blocks. Black verticals show the demarcation of distinct haplotype blocks at the start or and of a new pairwise IBD segment. The red and yellow shaded areas show examples of complete and incomplete founder IBD graphs, respectively. \textbf{(right)} A complete \textbf{(A)} and an incomplete \textbf{(B)} founder IBD graph. In cases where haplotypes were incomplete, the missing edges (i.e. the edge between Founder 1 and Founder 3) were filled in for downstream analysis.}
\label{fig:supfigure6}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/PCA_LD1.png}
\caption{\textbf{Principal Component Analysis of the 16 Founders, Tester, and the 344 MAGIC DH Lines.} 600K genotype data was filtered using an LD filter of 1 cM. The points in grey are MAGIC lines. The colored points show the tester, MBS847 (dark grey) and the 16 founders.}
\label{fig:supfigure7}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/Biogemma_Founder_to_DH_MAF.png}
\caption{\textbf{Minor Allele Frequencies of SNPs in the 16 Founders and the MAGIC lines.} The x-axis shows the minor allele frequency of 600K SNPs in the 16 founders. The y-axis shows the minor allele frequency of the corresponding SNPs in the MAGIC population.}
\label{fig:supfigure8}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/IBD_supplemental.png}
\caption{\textbf{Overlap between uncertainty in founder probabilities and pairwise Identity-By-Descent between founders.} \textbf{(top)} An example of founder probabilities across chromosome 10 for two MAGIC lines. The y-axis shows the probability of the line being derived from each of the 16 founders, shown in different colors. \textbf{(bottom)} Pairwise IBD state across chromosome 10 between two founders, F492 and A654, which correspond to uncertainty in founder probabilities above.}
\label{fig:supfigure5}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/nj_tree.png}
\caption{\textbf{Neighbor-Joining Tree of the 16 Founders and Tester using the 600K genotype data} A UPGMA dendrogram created using the TASSEL software \citep{Bradbury}.}
\label{fig:supfigure9}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/environment_QTL_by_method.png}
\caption{\textbf{Number of Environment-Specific QTL Found by Method} S: $GWAS_{SNP}$, F: $QTL_F$, H: $QTL_H$. Environment-specific QTL are QTL for a phenotype that were identified in one environment. QTL were called as found by multiple methods if their support intervals overlapped (see Methods).}
\label{fig:supfigure2}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/threshold_by_method_counts.png}
\caption{\textbf{Total Number of Environment-QTL Found By Method By Significance Threshold} S: $GWAS_{SNP}$, F: $QTL_F$, H: $QTL_H$. Environment-specific QTL are QTL for a phenotype that were identified in one environment. Color indicates new QTL found by raising the significance threshold from 1\% (red) to 5\% (green) to 10\% (blue).}
\label{fig:supfigure3}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/support_interval_cM.png}
\caption{\textbf{Size of QTL Support Intervals for Shared QTL by Method in Genetic Distance (cM)} 600K\_SNP: $GWAS_{SNP}$, Founder\_probs: $QTL_F$, Haplotype\_probs: $QTL_H$. Colors indicate individual Environment-QTL.}
\label{fig:supfigure15}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/support_interval_Mb.png}
\caption{\textbf{Size of QTL Support Intervals for Shared QTL by Method in Physical Distance (Mb)} 600K\_SNP: $GWAS_{SNP}$, Founder\_probs: $QTL_F$, Haplotype\_probs: $QTL_H$. Colors indicate individual Environment-QTL.}
\label{fig:supfigure16}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/mite_founder_haps.png}
\caption{\textbf{Haplotype View of the MITE insertion underlying \emph{vgt1} in the 16 Founders and tester, MBS847} Red blocks indicates the location of the MITE insertion. Founders are separated by $MITE^-$, $MITE^+$ Early, and $MITE^+$ Late based off of their allele at \emph{vgt1} and groupings based on the estimated founder effect size. Created using the haplostrips software \citep{Marnetto}}
\label{fig:supfigure14}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/vgt1_epistasis_scan.png}
\caption{\textbf{Genome Scan for Epistasis with \emph{vgt1}.} Red line indicates a Bonferroni 5\% significance threshold.}
\label{fig:supfigure12}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/MITE_only_DTA_manhattan.png}
\caption{\textbf{Manhattan plot Days to Anthesis BLUPs using only $MITE^+$ MAGIC lines and $QTL_F$} Red points indicate SNPs above a  5\% permutation significance threshold.}
\label{fig:supfigure13}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/all_chrom_founder_chi_sq.png}
\caption{\textbf{$\rchi^2$ Peaks for Deviation from Equal Founder Distribution by Chromosome} Red line indicates a 5\% significance threshold (-log10(p-value=8.82)) obtained from 100 simulated populations.}
\label{fig:supfigure11}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics[width=16cm,height=10cm]{figures/ft_overlap_enrichment_density.png}
\caption{\textbf{Enrichment of Flowering Time Genes in $\rchi^2$ Peaks} Density plot of 1,000 permutations of number of 904 randomly selected genes that overlap with $\rchi^2$ peaks for over- or under-representation of haplotypes in the MAGIC population. Red line indicates the actual number (284) of FT genes from the list of 904 that overlapped with $\rchi^2$ peaks.}
\label{fig:supfigure4}
\end{figure}


\endsupplement
\end{document} 
